### PR TITLE
[NCL-3738]: Remove tokens from logs - master

### PR DESCRIPTION
**Pull request by jbartece**

### CODE REVIEW

**Comment by thescouser89** [comment]
retest this please

File: pom.xml [hunk 1]
```diff
@@ -782,6 +782,12 @@
         <version>${version.catch-exception}</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
+        <groupId>org.slf4j</groupId>
+        <artifactId>slf4j-log4j12</artifactId>
```
Comment (matejonnet) [inline pom.xml]: Any particular reason to use log4j and not slf4j-simple which is used elsewhere?

Comment (jbartece) [reply pom.xml]: Yes, the log4j is needed by the OpenShift client library and without this dependency the tests were failing with ClassNotFoundError. Btw. The scope for it is only test.

**Comment by jbartece** [comment]
retest this please

**Pull request merged**